lib/app/modules/home/views/home_view.dart

```diff
@@ -1432,3 +1443,48 @@ class HomeView extends GetView<HomeController> {
     );
   }
 }
+
+Future<bool> showDeleteAlarmConfirmationPopupOnSwipe(
+  BuildContext context,
+) async {
+  // Return true if user confirms deletion, false if canceled
+
+  return await showDialog(
+        context: context,
+        builder: (BuildContext context) {
+          return AlertDialog(
+            title: const Text(
+              'Confirmation',
+              style: TextStyle(color: Colors.blueGrey),
+            ),
+            content: const Text(
+              'Are you sure you want to delete this alarm?',
+              style: TextStyle(
+                color: Colors.blueGrey,
+              ),
+            ),
+            actions: [
+              TextButton(
+                onPressed: () {
+                  Navigator.of(context).pop(false); // User canceled
+                },
+                child: const Text(
+                  'NO',
+                  style: TextStyle(color: Colors.green),
+                ),
+              ),
+              TextButton(
```

Comment:
Also, please make sure the buttons are consistent with the theme - A good example can be found in the confirmation to navigate away from the addOrUpdate screen